src/metatensor/models/alchemical_model/tests/test_functionality.py

```diff
@@ -29,10 +27,11 @@ def test_prediction_subset():
 
     alchemical_model = Model(capabilities, DEFAULT_HYPERS["model"]).to(torch.float64)
     structure = ase.Atoms("O2", positions=[[0.0, 0.0, 0.0], [0.0, 0.0, 1.0]])
-    nl_options = NeighborsListOptions(model_cutoff=5.0, full_list=True)
     system = rascaline.torch.systems_to_torch(structure)
-    nl = get_rascaline_neighbors_list(system, nl_options)
-    system.add_neighbors_list(nl_options, nl)
+    requested_neighbors_lists = alchemical_model.requested_neighbors_lists()
+    for nl_options in requested_neighbors_lists:
+        nl = get_rascaline_neighbors_list(system, nl_options)
+        system.add_neighbors_list(nl_options, nl)
```

Comment:
this is covered for you if you put your model inside `MetatensorAtomisticModel`

Reply:
Yes sure, but in the eval script and train script this needs to be done manually I guess. If this is only about a single test fix, I can wrap the model here in the `MetatensorAtomisticModel` class.

Reply:
Sorry, I misunderstood this code. This is good as is! I though you where using some internal code from `MetatensorAtomisticModel` to get the NL from a non-top level module, but this is not the case!

Reply:
So in the end, I can use a `MetatensorAtomisticModel` where it is possible (for checks), while keeping my own check and update NLs function, right?

Reply:
Done!